src/server/routes/index.ts

```diff
@@ -62,6 +63,9 @@ if (okta.enabled) {
   uncachedRoutes.use(oauth);
 }
 
+// email template routes
+uncachedRoutes.use(emailTemplates);
```

Comment:
As mentioned previously, these emails don't need to be uncached as they're all static html. So you can just `router.use`

```suggestion
router.use(emailTemplates);
```